tests/php/Core/Manifest/VersionProviderTest.php

```diff
@@ -2,14 +2,15 @@
 
 namespace SilverStripe\Core\Tests\Manifest;
 
-use Psr\SimpleCache\CacheInterface;
+use SebastianBergmann\Version;
+use SilverStripe\Dev\SapphireTest;
 use SilverStripe\Core\Config\Config;
 use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Core\Manifest\VersionProvider;
-use SilverStripe\Dev\SapphireTest;
 
 class VersionProviderTest extends SapphireTest
 {
+    const SEMVER_REGEX = '(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-(0|[1-9A-Za-z-][0-9A-Za-z-]*)(\.[0-9A-Za-z-]+)*)?(\+[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?';
```

Comment:
Please use composer/semver instead per my original request for this test (probably needs to be added as a dev dependency), there's no sense in us maintaining a se,very regex.

Reply:
done